src/main/resources/reference.conf

```diff
@@ -0,0 +1,20 @@
+jmt {
+  bwe {
+    send-side {
+      low-loss-threshold = 0.02
+      high-loss-threshold = 0.1
+      bitrate-threshold = 100 kbps
+
+      loss-experiment {
+        probability = 0
+        low-loss-threshold = 0.05
```

Comment:
Where do these values come from? The old code seems to default to 0.02, 0.1 and 100 kbps.

Reply:
Yup, I was playing with changing them to test and forgot to revert.